packages/preview/ansi-render-0.1.0/README.md

```diff
@@ -0,0 +1,23 @@
+# ANSI Escape Sequence Renderer
+
+This script provides a simple way to render text with ANSI escape sequences. It uses the `ansi_render` function from the `ansi_render.typ` module to render text with various formatting options, including font, font size and theme.
+
+contribution is welcomed!
+
+## Usage
+
+```typst
+#ansi_render(body, font: string, size: length, theme: terminal_themes.theme)
```

Comment:
it would be great if the usage example would provide full working code that can be copied. with an import from the preview namespace.